docs/mod-lists/skyblock-mod-list.md

```diff
@@ -81,6 +81,12 @@ Select your Minecraft version below to view compatible mods.
     ### SkyGuide
     - Download: https://modrinth.com/mod/skyguide/versions
     - Support: https://discord.gg/XtAuqsJWby
+
+    ### Skyblock Features (SBF)
+    Mod that adds features that are not commonly found or are paid in other mods but are very useful. Notably an Auction Flipper, highly accurate estimated item value, trash highlighting, glowing bestiary tracking, cooldown trackers, etc.
```

Comment:
>features that are not commonly found [...] in other mods

\*sigh\*

Disclaimer: Lists of mods within `()`s are not comprehensive, otherwise I would need to spend a week on finishing this list (instead of half of a day).

Features not found in the list(s) below are *assumed* to be unique to SBF until proven otherwise by others.

Let's start with `README.md`:
* Highlight Correct Livid `(Skytils)` `{it's a box, but close enough}`
* Crystal Hollows Treasure Chest Solver `(Skyblock Personalized)`
* 3 Weirdo Solver, Blaze Solver, Teleport Pad Solver `(Skytils, DSM)`
* Chronomotron & Ultrasequencer Solver `(NEU)`
* Crypt Display `(Skytils)`
* Hide Non-Starred Mobs `(Skytils)` `{why hide the entire mob? it's free combat experience}`
* Glowing Starred Mobs `(Skytils)`
* F3/M3 Fire freeze timer `(SkyHanni)`
* Highlight Dungeon Bats `(NEU)`
* Dungeon Party Display `(SkyBlock-Hud)`
* Dungeon Teammate Glowing `(SBA)`
* Dungeon Map with Player Heads And Names `(NEU, Dungeons Guide, BetterMap CT module)` `{adding player names just clutters the map anyways}`
* Dungeon Chest Profitx `{sic}` `(NEU)`
* Gemstone Profit Tracker `(SkyHanni)`
* Endernode Tracker `(SkyHanni)`
* Glacial Cave Treasure Tracker `(SkyHanni)`
* Commissions Tracker `(NEU, Skyblock Personalized)`
* Powder Mining Tracker `(NEU, SkyHanni)`
* Ghost Loot Tracker `(SkyHanni)`
* Metal Detector Solver `(NEU)`
* Crystal Hollows Map (Maps out explored caves) `(Skytils, Skyblock Personalized, SkyBlock-HUD, SkyGuide)`
* Small Items `(OverflowAnimations)`
* Glowing Items based on rarity `(SBA)`
* Bestiary Mob Tracker (can glow any mob) `(NwjnAddons CT module)`
* Glowing Zealots `(SBA)` `[Change Zealot Color]`
* Highlight Slayers `(DSM)`
* Ad Blocker (Hides people advertising in chat) `(Chatting)`
* Helpful Auction Guis (for flipping, selling, and buying) `(PartlySaneSkies, Skytils, NEU)` `{Skytils + NEU to prevent overselling/underselling, PSS for GUIs. All three display LBIN data to some extent; "suggesting" sell prices through an arbitrary one-liner is not the way to go}`
* Better Party finder `(Dungeons Guide, Cowlection)`
* Show Extra Profile Info (Shows networth, skill avg and discord when looking at someones profile) `(NEU)` `{adds "/pv" command block button + if you want someone's discord tag so much, "Open in SkyCrypt" from NEU's own profile viewer}`
* Fairy Soul Helper `(NEU)`
* Diana Mythological Helper `(SkyHanni, Soopy)`
* Endernode Highlighter `(NEU)`
* 1.12 Crop Hitbox `(Patcher)`
* Highlight Glowing Mushrooms `(NEU)`
* Lowest BIN Price `(NEU)`
* Average BIN Price `(NEU)`
* Estimated Item Price `(SkyHanni)`
* Item Price Paid `(Skytils)`
* Enigma Soul Waypoints `(SkyHanni)`
* Mirror verse puzzle solvers `(SkyHanni)`
* Minikloon Hacking Solver `(SkyHanni)`
* **/sky** and **/pv** commands `(NEU)`

Now let's comb through `Config.java`:
* Customizable Sidebar `(VanillaHUD)`
* Salvage Protection `(Skytils)`
* Quick Close Chest `(Dungeons Guide)`
* Blessings Viewer `(Cowlection)`
* Hide Other Player Gifts `(SBA)`
* Highlight Mini-bosses `(Skytils)` `{it's a box, but close enough}`
* Highlight Voidgloom Stage Colors `(Skytils)`
* Slayer Timer `(Skytils)`
* Beacon Highlight Color `(Skytils)`
* Day Tracker `(SkyblockPersonalized)`
* `{Health & Mana Bars}` Category `(SBA + Apec)`
* Highlight Odanta `{sic}` `(SkyHanni)`
* Relic Helper `(Skytils)`
* Dynamic Fullbright `(Patcher)` `{let's be honest, name one skyblock player who willingly plays without persistent fullbright}`
* Hide Players Near NPC's `{sic}` `(SBA)`
* Hide Arrows `(Patcher)`
* Treecapitator Cooldown `(NEU, SBA, PartlySaneSkies)`
* On-Screen Clock `(EvergreenHUD, SkyHanni)`
* Jerry Timer `(Skytils {for counting upwards}, Skyblock Personalized {for counting downwards})`
* Use Smooth Font `{use texture pack, also an outright misnomer for folks who don't use Mojang's font for Minecraft}`
* Player Cape `(Mixmetica, Synthesis)`
* Crop Counter `(NEU)`
* Composter Overlay `(SkyHanni)`
* Trevor The Trapper Helper `(SBA, SkyHanni)`
* Auto Accept Reparty `(Skytils)`
* Auto Reparty `(Skytils)`
* Dwarven Mines Map `(SkyGuide)`
* Show teleport overlay `(NEU)`
* Cooldown Tracker `(SkyHanni)`
* Chat Timestamps `(Chatting, Patcher)`
* {Better Trading} category `(NEU)`
* Final Destination Armor Display `(SBA)`
* Show Advanced Dragon Hitbox `(PolyHitboxes)`
* Garden Visitor Overlay `(SkyHanni)`
* Hide Air Display `(Patcher)` `[Clean View]`
* Hide Geyser Particles `(OptiFine)` `[Video Settings]` `{or just use hypixel's /particlequality command}`
* No Fire `(Patcher)`
* No Hurtcam `(Skytils, BetterHurtCam)`
* Low Arrow Notification `(Skytils)` `[Custom Notifications]`
* Hide Anita Artifact Notification `(Skytils)` `[Spam Hider]`
* Hide Command Cooldown Warning `(Skytils)`
* Toggle Sprint `(PolySprint)`

Do what you will with this information.